velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -813,6 +813,11 @@ TypePtr ReaderBase::convertType(
       case thrift::Type::type::INT32:
         return INTEGER();
       case thrift::Type::type::INT64:
+        // For Int64 Timestamp in nano precision
```

Comment:
> in nano precision

nit: is other precision other than nano possible?

Reply:
No, other precisions handle separately: see https://github.com/facebookincubator/velox/issues/11600#issuecomment-2512065714